packages/amplify-provider-awscloudformation/src/utils/sandbox-mode-helpers.ts

```diff
@@ -15,7 +15,7 @@ export async function showSandboxModePrompts(context: $TSContext): Promise<any>
     printer.info(
```

Comment:
nit: could use `printer.warn` here. If you switch it, remove the leading ⚠️ as this will be prepended by `warn` internally